### PR TITLE
[WAITING] Feature/set register to a bulk message

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
## Checklist:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

File: contracts/ExtendedAdvertisement.sol [hunk 1]
```diff
@@ -28,6 +28,10 @@ contract ExtendedAdvertisement is BaseAdvertisement, Whitelist, Signature {
             string endPoint
```
Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 113 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 113 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 113 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 4 failing
```
```diff 
+ 83 passing
```
#### Test coverage: 
67.8 %

Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 113 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline contracts/ExtendedAdvertisement.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 113 passing
```
#### Test coverage: 
83.43 %

**Comment by GustavoRSSilva** [comment]
Closing this Pr, it will be merged into a single one.

**Pull request closed without merging**